Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -1900,6 +1925,10 @@ public function freezePackageReflection($packageKey): void
 
     /**
      * Filter an array of entries where keys are class names by being in the given package namespace.
+     *
+     * @param (int|string) $packageKey
```

Comment:
Where the parentheses suggested by PhpStorm? I'd write it in "plain PHP syntax"…

```suggestion
     * @param int|string $packageKey
```

Same for the other places where this is used (reading the diff "backwards", so…)

Reply:
I tried letting Psalm resolve a few issues which went completely havoc and apparently I forgot to revert these changes at some places. This was not intentional.